src/Symfony/Component/Serializer/Tests/NameConverter/AdvancedNameConverterTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\NameConverter;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Serializer\NameConverter\AdvancedNameConverterInterface;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class AdvancedNameConverterTest extends TestCase
+{
+    public function testImplem()
+    {
+        $p = $this->prophesize(AdvancedNameConverterInterface::class);
```

Comment:
Still WIP I guess 😄 
Also we don't use prophecy in symfony core test suite.

Reply:
Oops, I don't want to commit this file (yes I know for prophecy :(, it would be the perfect fit here).